lib/audit/orphanedIncludes.js

```diff
@@ -0,0 +1,95 @@
+const fs = require('fs');
+const glob = require('tiny-glob');
+const path = require('path');
+
+const logger = require('../logger');
+
+const includes = /{%-?\ include\ (['"])([\w/]*)\1/g;
+const variableInclude = /{%-?\ include\ ([\w]+)/;
+
+// Filter for {app,modules}/ top level directories.
+// This is necessary because tiny-glob doesn't return files from all subdirectories: https://github.com/terkelg/tiny-glob/issues/28
+const filterAppModules = files => files.filter(file => file.match(/^(app|modules)\//));
+
+const normalizePath = (partialName, includedFrom) => {
+    // App directory.
+    if (includedFrom.startsWith('app/')) {
+        return findExistingFile([
+            `app/views/partials/${partialName}.liquid`,
+            `app/views/partials/_${partialName}.liquid`
+        ]);
+    }
+
+    // Modules.
+    // Get module/[moduleName].
+    const includedFromPath = path.join(...includedFrom.split(path.sep).slice(0, 2));
+    return findExistingFile([
+        `${includedFromPath}/private/views/partials/${partialName}.liquid`,
+        `${includedFromPath}/private/views/partials/_${partialName}.liquid`,
+        `${includedFromPath}/public/views/partials/${partialName}.liquid`,
+        `${includedFromPath}/public/views/partials/_${partialName}.liquid`
+    ]);
+};
+
+const findExistingFile = files => {
+    const matchingFiles = files.filter(file => fs.existsSync(file));
+    if (matchingFiles.length > 0) {
+        return matchingFiles[0];
+    }
+
+    return "";
+}
+
+const findPartials = async () => {
+    let foundVariableInclude = false;
+    let foundVariableIncludeFile = "";
+
+    const files = filterAppModules(await glob('**/*.liquid', { filesOnly: true }));
```

Comment:
This probably can be done without the filterAppModules hack now, since tiny-glob is gone :)